recipes/omnixai/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "omnixai" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omnixai-{{ version }}.tar.gz
+  sha256: 0940a160be9265d9bfc00f3a6a7019cc05e40afcb77b0ad489ee61114b9f989d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - dill
+    - hnswlib >=0.5.0
+    - ipython
+    - lime
+    - matplotlib-base
+    - numpy >=1.17
+    - packaging
+    - pandas >=1.1.0
+    - pillow <10.0
+    - salib
+    - scikit-image >=0.17.2
+    - scikit-learn >=0.24,<1.2
+    - scipy >=1.5.0
+    - shap >=0.40.0
+    - statsmodels >=0.10.1
+    - tabulate
+    - tqdm
+    - wheel
+
+test:
+  imports:
+    - docs
```

Comment:
This package should not install a module called `docs`.

Reply:
Thank you for point this out. Let me also check with upstream on why docs is being packaged in the source. 

My guess is: it should not be a part of the source package on PyPI (probably they did not specify this in a MANIFEST.in file).